utilities/src/main/java/io/syndesis/qe/resource/impl/Ops.java

```diff
@@ -0,0 +1,204 @@
+
+    @Override
+    public void deploy() {
+        final NamespacedOpenShiftClient ocp = getMonitoringNamespace();
+        log.info("Installing monitoring applications");
+        if (!isDeployed()) {
+            try {
+                Process p;
+                if (!new File(TMP_FOLDER).exists()) {
+                    p = new ProcessBuilder()
+                        .directory(new File("/tmp"))
+                        .redirectOutput(new File("/tmp/gitout"))
+                        .command("git", "clone", "--depth", "1", "--branch", TestConfiguration.appMonitoringVersion(),
+                                 "https://github.com/integr8ly/application-monitoring-operator")
+                        .start();
+                    p.waitFor();
+                    //Use oc used by the testsuite
+                    Path scriptPath = Paths.get(TMP_FOLDER, "scripts", "install.sh");
+                    String script = IOUtils.toString(Files.newReader(scriptPath.toFile(), Charset.defaultCharset()));
+                    String ocReplaced = script.replaceAll("oc", OpenShifts.getBinaryPath());
+                    Files.write(ocReplaced.getBytes(Charset.defaultCharset()), scriptPath.toFile());
+                }
+                //Oc can be logged in to only one cluster at a time, this ensures the OC is really logged in
+                new ProcessBuilder().command(
+                    OpenShifts.getBinaryPath(),
+                    "login",
+                    TestConfiguration.openShiftUrl(),
+                    "-u", TestConfiguration.adminUsername(),
+                    "-p", TestConfiguration.adminPassword()
+                ).start().waitFor();
+                p = new ProcessBuilder()
+                    .directory(new File(TMP_FOLDER))
+                    .redirectOutput(new File("/tmp/out"))
+                    .command("make", "cluster/install")
+                    .start();
+                int ret = p.waitFor();
+                if (ret != 0) {
+                    log.error("Installation process finished with code {}", ret);
+                    InfraFail.fail("Application monitoring stack installation failed with following log: {}",
+                                   IOUtils.toString(new URL("/tmp/out").openStream(), Charset.defaultCharset()));
+                }
+                OpenShiftWaitUtils.waitFor(() -> ocp.routes().list().getItems().size() >= 3);
+            } catch (IOException | InterruptedException | TimeoutException e) {
+                log.error("Monitoring applications deployment failed", e);
+                Assert.fail("Monitoring applications deployment failed \n" + e.getMessage());
+            }
+        }
+        OpenShiftUtils.binary().execute("adm", "policy", "add-cluster-role-to-user", "view", "system:anonymous");
+        OpenShiftUtils.binary()
+            .execute("adm", "policy", "add-cluster-role-to-user", "cluster-admin", String.format("system:serviceaccount:%s:prometheus-operator",
```

Comment:
I know it's easier to do it this way, but it can be done using `OpenShiftUtils.getInstance().rbac().clusterRoleBindings().createOrReplaceWithNew()...`, there is an example of this in Kafka class

Reply:
You are right, no longer necesary